doc/admin-guide/files/records.config.en.rst

```diff
@@ -2207,8 +2207,8 @@ Cache Control
 .. ts:cv:: CONFIG proxy.config.http.cache.ignore_authentication INT 0
    :overridable:
 
-   When enabled (``1``), |TS| ignores ``WWW-Authentication`` headers in responses ``WWW-Authentication`` headers are removed and
-   not cached.
+   When enabled (``1``), |TS| ignores ``WWW-Authentication`` headers in
+   responses and the responses are cached.
```

Comment:
We discussed this offline, but just to record it here: It would be explicitly list the pragma and cache-control directives that are ignored for `proxy.config.http.cache.ignore_client_no_cache` and `proxy.config.http.cache.ignore_server_no_cache` so it's clearer to users what is being ignored when those features are enabled.

Reply:
Updated the doc for these. Thanks!

Reply:
Looks good.